app/fissile.go

```diff
@@ -601,6 +584,32 @@ func (f *Fissile) GetDiffConfigurationBases(releasePaths []string, cacheDir stri
 	return getDiffsFromReleases(f.releases)
 }
 
+// Since each job processes only its own spec, inject the hcf-release's
+// patches-properties pseudo-job's spec into all the other jobs.
+func (f *Fissile) injectPatchPropertiesJobSpec() error {
+	var patchPropertiesJob *model.Job
+	for _, release := range f.releases {
+		if release.Name == "hcf" {
```

Comment:
Where the patch properties come from needs to be configurable, since this might _not_ be using `hcf` :disappointed: (e.g. `cf-solo` does its own thing)